temporal-sdk/src/main/java/io/temporal/internal/worker/WorkflowTaskHandler.java

```diff
@@ -76,6 +80,13 @@ public boolean isCompletionCommand() {
       return completionCommand;
     }
 
+    public Functions.Proc1<Long> getEventIdSetHandle() {
```

Comment:
I have a slight preference for requiring this field to be non-null and passing a do-nothing value in places where we don't want to do the reset. Minor though.